workers/reconciliationworker.py

```diff
@@ -48,21 +47,39 @@ def _perform_reconciliation(self, user_api, marketplace_api):
         for user in stripe_users:
 
             email = user.email
-            ebsAccountNumber = entitlements.get_ebs_account_number(user.id)
+            model_customer_id = entitlements.get_web_customer_id(user.id)
             logger.debug(
-                "Database returned %s account number for %s", str(ebsAccountNumber), user.username
+                "Database returned %s customer id for %s", str(model_customer_id), user.username
             )
 
-            # go to user api if no ebsAccountNumber is found
-            if ebsAccountNumber is None:
-                logger.debug("Looking up ebsAccountNumber for email %s", email)
-                ebsAccountNumber = user_api.lookup_customer_id(email)
-                logger.debug("Found %s number for %s", str(ebsAccountNumber), user.username)
-                if ebsAccountNumber:
-                    entitlements.save_ebs_account_number(user, ebsAccountNumber)
+            # check against user api
+            logger.debug("Looking up webCustomerId for email %s", email)
```

Comment:
nit: A little redundant. You can just add the email to the following log statement